apps/calendar/style/modify_account_view.css

```diff
@@ -24,7 +24,7 @@
 }
 
 #modify-account-view.in-progress .during-progress > div {
-  display: inline-block;
+  display: block;
```

Comment:
So what's your suggestion ? If I use `inline-block` the margin looks like shorter ? and If I edit the margin it will effect the whole system. Do you have any better idea ?